src/README.rst

```diff
@@ -16,6 +16,15 @@ To get started, after installation run the following:
 Change Log
 ==========
 
+3.2.0
+-----
+
+- Update to 6.1 Service Fabric runtime
```

Comment:
Merge from master, new version is `4.0.0`

Reply:
done.